ui/src/test/SortCursor.test.js

```diff
@@ -0,0 +1,45 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import sinon from 'sinon';
+import SortCursor from "../Components/Dashboard/Table/TableHeader/SortCursor";
+import { TABLE } from "../Constant/constant";
+
+describe("SortCursor Component", () => {
+
+    it("Should render the SortCursor Component with Ascending cursor", () => {
+      render(<SortCursor />)
+
+      const cursor = screen.getByTitle(TABLE.ASCENDING)
+      expect(cursor).toBeInTheDocument(); 
+    })
+
+    it("Should render the SortCursor Component with Descending cursor", () => {
+      render(<SortCursor />)
+
+      const cursor = screen.getByTitle(TABLE.DESCENDING)
+      expect(cursor).toBeInTheDocument();
+       
+    })
+
+    it("Should fire event on click ascending", async () => {
+      const mockFn = sinon.spy();
```

Comment:
Can we use Jest and drop sinon, please? Reason: We have Jest already. Let me know if you prefer Sinon over Jest

Reply:
Ok I'll recheck this.

Reply:
Possilble. removing sinon